src/main/java/com/keyboardbuilder/enterprise/services/IBuildService.java

```diff
@@ -5,13 +5,13 @@
 import java.util.List;
 
 public interface IBuildService {
-    Build fetchById(int id);
+    Build fetchById(int id) throws Exception;
 
-    void delete(int id) throw Exception;
+    void delete(int id) throws Exception;
```

Comment:
Fixed build error, since throw must be throws

Reply:
Good.  throw will not work in this case.